docs/_data/components/schemas/array-max-monitored-elements.yaml

```diff
@@ -11,17 +11,27 @@ forecast-limits-snapshot:
       description: Set of forecast limits
       items:
         $ref: 'array-max-forecast-periods.yaml#/forecast-limit-item'
+  required:
+    - snapshot-header
+    - limits
 
-forecast-limits-slim-snapshot:
-  type: object
-  description: A snapshot of the forecast for a monitoring set.
-  properties:
+forecast-limits-snapshot-slim:
+  <<: *forecast-limit-snapshot
+  description: A slim snapshot of the forecast for a monitoring set.
+  properties: &forecast-limit-snapshot-slim-props
     snapshot-header:
       $ref: ./headers.yaml#/forecast-snapshot-slim-header
     limits:
       <<: *max
       items:
         $ref: 'array-max-forecast-periods.yaml#/resource-forecast-proposal-slim'
+forecast-limits-snapshot-slim-inputs-used:
+  <<: *forecast-limit-snapshot
+  description: |
+    A slim snapshot of the forecast for a monitoring set, including the inputs
+    that were used to determine the continuous limits.
```

Comment:
@getorymckeag Please take particular note of this: *continuous limits*. The issue is that different providers can potentially set continuous versus emergency limits, so a single input for each period's rating set might be insufficient. Since this is about creating situational awareness in interop, I felt the compromise of only guarantee the input was associated with the continuous limit was worthwhile.

Reply:
I think this is fine.  I am having trouble imaging a scenario where the inputs are actually different.  It would be kind of strange to have a weather measurement that is used to generate an emergency rating but NOT a continuous rating.  While the inverse seems at least possible (maybe ignore some more "exotic" cooling factor?), the difference for emergency limits and continuous is really an internal of the TO's rating methodology, and I would think is something market monitors could work with the TO to simply rederive from the available continuous input + equipment characteristics if they really wanted to.